src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -68,8 +69,8 @@ export const ChannelView: React.FC = () => {
   const followedChannels = usePersonalDataStore((state) => state.followedChannels)
   const updateChannelFollowing = usePersonalDataStore((state) => state.actions.updateChannelFollowing)
   const [isFollowing, setFollowing] = useState<boolean>()
-  const [currentVideosTab, setCurrentVideosTab] = useState(0)
-  const currentTabName = TABS[currentVideosTab]
+  const [currentTabIndex, setCurrentTabIndex] = useState(0)
```

Comment:
Why do we need this index? I think we can just use the state from the search params?

Reply:
I think thats right, removed it